Patch origin: in-house
Patch status: need to be submitted to upstream

--- a/tests/setup
+++ b/tests/setup
@@ -6,7 +6,7 @@
 
 JQTESTDIR=$(cd "$(dirname "$0")" && pwd)
 JQBASEDIR=$JQTESTDIR/..
-JQ=$JQBASEDIR/jq
+JQ=${JQ:-$JQBASEDIR/jq}
 
 if [ -z "${NO_VALGRIND-}" ] && which valgrind > /dev/null; then
     VALGRIND="valgrind --error-exitcode=1 --leak-check=full \
